src/DIRAC/ConfigurationSystem/Client/Utilities.py

```diff
@@ -139,12 +140,14 @@ def getGridCEs(vo, bdiiInfo=None, ceBlackList=None, hostURL=None, glue2=False):
     return result
 
 
-def getSiteUpdates(vo, bdiiInfo=None, log=None, glue2=True):
+def getSiteUpdates(vo, bdiiInfo=None, log=None, glue2=True, onecore=False):
     """Get all the necessary updates for the already defined sites and CEs
 
     :param str vo: VO name
     :param dict bdiiInfo: information from DBII
     :param object log: logger
+    :param bool glue2: whether to use glue2 information
```

Comment:
Since `glue2` option is removed from later branches, I would not add this line here.

Reply:
Yes, dropped this param doc